libs/components/src/lib/rich-text-editor/rich-text-editor.ts

```diff
@@ -33,10 +33,36 @@ export class RichTextEditor extends VividElement {
 		return this.shadowRoot!.querySelector('#editor') as HTMLElement;
 	}
 
-	get selection(): RichTextEditorSelection | undefined {
-		return this.#editor?.selection();
+	@attr({ converter: nullableNumberConverter, attribute: 'selection-start' })
+	selectionStart: number | null = null;
+	selectionStartChanged() {
```

Comment:
It's needs the despised `@internal` or it will be public
```suggestion
	selectionStartChanged() {
```